TAScheduler/Managment/UserManagement.py

```diff
@@ -14,8 +19,37 @@ def __init__(self, authID=""):
     # User Address(in) - Address of the user
     # User Password(in) - Password of the user
     # User Type(in) - Type of the user
-    def createUser(self):
-        pass
+    def createUser(self, Id, name, contact, address, password, email,usertype):
+        if(not(isinstance(Id, int))):
+            raise TypeError("Id entered is not of type int")
+        if(not(isinstance(name, str))):
+            raise TypeError("Name entered is not of type str")
+        if(not(isinstance(contact, str))):
+            raise TypeError("Contact entered is not of type str")
+        if(not(isinstance(email, str))):
+            raise TypeError("Email entered is not of type str")
+        if(not(isinstance(address, str))):
+            raise TypeError("Address entered is not of type str")
+        if(not(isinstance(password, str))):
+            raise TypeError("Address entered is not of type str")
+        if(not(isinstance(usertype, int))):
```

Comment:
userType should be a string, not an int. The options for this are "SUPERVISOR", "INSTRUCTOR", and "TA".

Reply:
I believe we no longer planned to store a value for SSN, as indicated by the Database Design document. I've attached that here for ease of access
[Database Diagram.pdf](https://github.com/YazanSalem/CS361Project/files/7688743/Database.Diagram.pdf)
.

Reply:
```suggestion
        if not (userType in ["SUPERVISOR", "INSTRUCTOR", "TA"]:
```
Potential solution to the if statement portion.